packages/@lwc/engine-server/src/__tests__/fixtures/export-default-component/modules/x/cmp/cmp.html

```diff
@@ -0,0 +1,2 @@
+<template>
+</template>
```

Comment:
```suggestion
<template>
  yolo
</template>
```

Putting something inside of here actually susses out the difference between `export default Component` and `export { Component as default }`. Without it, we just render an empty `<template shadowrootmode="open">` (probably due to the default `.html` file).